test/functional/wallet_taproot.py

```diff
@@ -300,7 +304,7 @@ def do_test_sendtoaddress(self, comment, pattern, privmap, treefn, keys_pay, key
             test_balance = int(rpc_online.getbalance() * 100000000)
             ret_amnt = random.randrange(100000, test_balance)
             # Increase fee_rate to compensate for the wallet's inability to estimate fees for script path spends.
-            res = rpc_online.sendtoaddress(address=self.boring.getnewaddress(), amount=Decimal(ret_amnt) / 100000000, subtractfeefromamount=True, fee_rate=200)
+            res = rpc_online.sendtoaddress(address=self.boring.getnewaddress(), amount=Decimal(ret_amnt) / 100000000, subtractfeefromamount=True, fee_rate=500)
```

Comment:
Why are you changing the fee rate?

Reply:
If I didn't bump the fee rate I would run into the two error codes mentioned in the PR message:
```
Had to bump fee rate to resolve assertions in sendtoaddress and psbt methods:

    code -5: assert rpc_online.gettransaction(txid)["confirmations"] > 0
    code -26: min relay fee not met
```

The previous `fee_rate = 200` limit seems to be sufficient up until `n ~ 500`. I assume this is because the script size increases with higher numbers of `n`.

Reply:
Fee rate is the fee per vbyte. But this may be related to #26573